Resources/Prototypes/DeltaV/Entities/Mobs/Species/chitinid.yml

```diff
@@ -0,0 +1,161 @@
+- type: entity
+  save: false
+  name: Urist McAnt
+  parent: BaseMobSpeciesOrganic
+  id: BaseMobChitinid
+  abstract: true
+  components:
+  - type: HumanoidAppearance
+    species: Chitinid
+    hideLayersOnEquip:
+    - HeadTop
```

Comment:
These are currently missing a Language, such that one would have to be made for them.

Reply:
We can just straight up copy the language for Kidan in Paradise, which is Paradise's ant species:
```dm
/datum/language/kidan
	name = "Chittin"
	desc = "The noise made by rubbing its antennae together is actually a complex form of communication for Kidan."
	speech_verb = "rubs their antennae together"
	ask_verb = "rubs their antennae together"
	exclaim_verbs = list("rubs their antennae together")
	colour = "kidan"
	key = "4"
	flags = RESTRICTED | WHITELISTED
	syllables = list("click","clack")
```
https://github.com/ParadiseSS13/Paradise/blob/e8f78062aae5fc839f6a9715e5d3f18723b80ad4/code/modules/mob/language.dm#L294-L303

Reply:
![image](https://github.com/user-attachments/assets/615969d8-56a6-4397-bcd4-0a14ad235e38)
Aurora *also* has an ant language.

Reply:
Vaurca's syllables are definitely better